### PR TITLE
Update security-guidelines.md

### PR DESCRIPTION
**Pull request by kmsft**
Added info on Windows Autopatch.

### CODE REVIEW

File: windows-365/enterprise/security-guidelines.md [hunk 1]
```diff
@@ -32,16 +32,18 @@ ms.collection: M365-identity-device-management
 
 To help improve security for your Cloud PCs, consider the following general guidelines:
 
-1. Apply Conditional Access policies to control the devices and apps that can connect to your email and company resources. Use Conditional Access to secure access end user access to Windows 365.Specifically, considering using Azure Active Directory (Azure AD) Multi-Factor Authentication to authenticate users. For more information, see [What is Conditional Access in Azure Active Directory?](/azure/active-directory/conditional-access/overview)
+1. Apply Conditional Access policies to control the devices and apps that can connect to your email and company resources. Use Conditional Access to secure access end user access to Windows 365. Specifically, consider using Azure Active Directory (Azure AD) Multi-Factor Authentication to authenticate users. For more information, see [What is Conditional Access in Azure Active Directory?](/azure/active-directory/conditional-access/overview) and how to use [Conditional Access to require Multi-Factor Authentication](/azure/active-directory/conditional-access/howto-conditional-access-policy-all-users-mfa)
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/security-guidelines.md:35]:
```suggestion
1. Apply Conditional Access policies to control the devices and apps that can connect to your email and company resources. Use Conditional Access to secure end-user access to Windows 365. Specifically, consider using Azure Active Directory (Azure AD) Multi-Factor Authentication to authenticate users. For more information, see [What is Conditional Access in Azure Active Directory?](/azure/active-directory/conditional-access/overview) and how to use [Conditional Access to require Multi-Factor Authentication](/azure/active-directory/conditional-access/howto-conditional-access-policy-all-users-mfa).
```

File: windows-365/enterprise/security-guidelines.md [hunk 2]
```diff
@@ -32,16 +32,18 @@ ms.collection: M365-identity-device-management
...
-6. Windows 365 integrates with Microsoft Defender for Endpoint. Security and endpoint admins can work together to manage their Cloud PC environment just like they manage a physical endpoint. If subscribed, Cloud PCs will:
+4. One of the most important elements of device security is OS updates. These updates make sure that devices stay up-to-date and secure while delivering new features and defenses against vulnerabilities. For Cloud PCs, options include:
+   - **Windows Autopatch** is a cloud service that automates updates for Windows, Microsoft 365 Apps for enterprise, Microsoft Edge, and Microsoft Teams on both physical and virtual devices. For more information, see [What is Windows Autopatch?](/windows/deployment/windows-autopatch/overview/windows-autopatch-overview) and the [Windows Autopatch FAQ](https://go.microsoft.com/fwlink/?linkid=2200228). Tor turn on Windows Autopatch, [edit your provisioning policy](/en-us/windows-365/enterprise/edit-provisioning-policy)
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/security-guidelines.md:43]:
```suggestion
4. One of the most important elements of device security is OS updates. These updates make sure that devices stay up-to-date and secure while delivering new features and defenses against vulnerabilities. For cloud PCs, options include:
   - **Windows Autopatch** is a cloud service that automates updates for Windows, Microsoft 365 Apps for Enterprise, Microsoft Edge, and Microsoft Teams on both physical and virtual devices. For more information, see [What is Windows Autopatch?](/windows/deployment/windows-autopatch/overview/windows-autopatch-overview) and the [Windows Autopatch FAQ](https://go.microsoft.com/fwlink/?linkid=2200228). Tor turn on Windows Autopatch, [edit your provisioning policy](/windows-365/enterprise/edit-provisioning-policy).
```

Comment (JohanFreelancer9) [inline windows-365/enterprise/security-guidelines.md:43]:
```suggestion
   - **Windows Autopatch** is a cloud service that automates updates for Windows, Microsoft 365 Apps for Enterprise, Microsoft Edge, and Microsoft Teams on both physical and virtual devices. For more information, see [What is Windows Autopatch?](/windows/deployment/windows-autopatch/overview/windows-autopatch-overview) and the [Windows Autopatch FAQ](https://go.microsoft.com/fwlink/?linkid=2200228). Tor turn on Windows Autopatch, [edit your provisioning policy](/windows-365/enterprise/edit-provisioning-policy)
```

File: windows-365/enterprise/security-guidelines.md [hunk 3]
```diff
@@ -32,16 +32,18 @@ ms.collection: M365-identity-device-management
...
+   - **Endpoint Manager** can be used by IT admins to configure Intune Windows 10/11 update rings and policies for Windows Update for Business. For more information, see [Manage Windows 10/11 software updates in Intune](/mem/intune/protect/windows-update-for-business-configure).  
+6. By default Windows 365 Enterprise, end users aren't administrators of their Cloud PCs. This policy aligns with Windows 10/11 security guidance. For more information about this guidance, see [Local Accounts](/windows/security/identity-protection/access-control/local-accounts#sec-restrict-protect-accounts) in the Windows documentation.
+7. Windows 365 integrates with Microsoft Defender for Endpoint. Security and endpoint admins can work together to manage their Cloud PC environment just like they manage a physical endpoint. If subscribed, Cloud PCs will:
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/security-guidelines.md:46]:
```suggestion
6. By default, in Windows 365 Enterprise end users aren't administrators of their cloud PCs. This policy aligns with Windows 10/11 security guidance. For more information about this guidance, see [Local Accounts](/windows/security/identity-protection/access-control/local-accounts#sec-restrict-protect-accounts).
7. Windows 365 integrates with Microsoft Defender for Endpoint. Security and endpoint admins can work together to manage their cloud PC environment just like they manage a physical endpoint. If subscribed, cloud PCs will:
```

File: windows-365/enterprise/security-guidelines.md [hunk 4]
```diff
@@ -32,16 +32,18 @@ ms.collection: M365-identity-device-management
...
+   - **Windows Autopatch** is a cloud service that automates updates for Windows, Microsoft 365 Apps for enterprise, Microsoft Edge, and Microsoft Teams on both physical and virtual devices. For more information, see [What is Windows Autopatch?](/windows/deployment/windows-autopatch/overview/windows-autopatch-overview) and the [Windows Autopatch FAQ](https://go.microsoft.com/fwlink/?linkid=2200228). Tor turn on Windows Autopatch, [edit your provisioning policy](/en-us/windows-365/enterprise/edit-provisioning-policy)
+   - **Endpoint Manager** can be used by IT admins to configure Intune Windows 10/11 update rings and policies for Windows Update for Business. For more information, see [Manage Windows 10/11 software updates in Intune](/mem/intune/protect/windows-update-for-business-configure).  
+6. By default Windows 365 Enterprise, end users aren't administrators of their Cloud PCs. This policy aligns with Windows 10/11 security guidance. For more information about this guidance, see [Local Accounts](/windows/security/identity-protection/access-control/local-accounts#sec-restrict-protect-accounts) in the Windows documentation.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/security-guidelines.md:45]:
```suggestion
6. By default, in Windows 365 Enterprise end users aren't administrators of their Cloud PCs. This policy aligns with Windows 10/11 security guidance. For more information about this guidance, see [Local Accounts](/windows/security/identity-protection/access-control/local-accounts#sec-restrict-protect-accounts).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
@kmsft Please recreate this PR in the private repo. Closing.

**Pull request closed without merging**